appserver/pom.xml

```diff
@@ -396,8 +396,8 @@
                             </artifact>
                             <nonFinal>false</nonFinal>
                             <jarType>api</jarType>
-                            <specVersion>1.2</specVersion>
-                            <specImplVersion>1.2.1</specImplVersion>
+                            <specVersion>${jstl-api.version}</specVersion>
```

Comment:
I think spec version should stay at 1.2. 1.2.2 seems to be the implementation artefact of the 1.2 spec.